### ISSUE TITLE
Error logging in

### ISSUE THREAD

**Issue by melissasage** [NicolasConstant/sengi#198]
I'm using the latest snap image and when I attempt to login with my accounts--specifically when I hit the "Authorize App" button--I get the following error in my console:

```
{ Error: ENOENT, dist/404.html not found in /snap/sengi/25/resources/app.asar
    at createError (ELECTRON_ASAR.js:131:17)
    at Object.fs.readFile (ELECTRON_ASAR.js:548:23)
    at /snap/sengi/25/resources/app.asar/main-electron.js:182:16
    at process.nextTick (ELECTRON_ASAR.js:21:28)
    at process._tickCallback (internal/process/next_tick.js:61:11) code: 'ENOENT', errno: -2 }
```

I'm on Manjaro Linux, if that makes any difference, and it's happened with several Mastodon instances.

**Comment by NicolasConstant** [comment]
Hi! 
Sorry to hear that!
You're using the snap version?

**Comment by melissasage** [comment]
Yes I am.

**Comment by NicolasConstant** [comment]
Thanks! I'll have a look to it. :)

**Comment by melissasage** [comment]
Thank you! Let me know if there's any other debugging information I can help provide.

**Comment by NicolasConstant** [comment]
So I broke the login process in Electron when making the Sengi Docker image...
It's now fixed (in 0.19.1), in order to add new accounts, you'll need to clear all stored data by following those steps:
* Settings/Clear all data
* Try to login again

Then it should work as expected. :)
Thanks for your help!
